test/Conversion/amd/decompose-unsupported-conversions-gfx9.mlir

```diff
@@ -0,0 +1,33 @@
+// RUN: triton-opt %s --split-input-file --decompose-unsupported-amd-conversions=arch=gfx942 | FileCheck %s
+
+// CHECK: #[[DST_ENC:.+]] = #triton_gpu.blocked<{{.*}}>
+// CHECK: #[[SRC_ENC:.+]] = #triton_gpu.amd_mfma<{{.*}}>
```

Comment:
Wanna specify out some different pieces between `SRC_ENC` and `TMP_ENC` and change the `CHECK` here to `CHECK-DAG`? Right now it relies on solely the ordering to not break..